lib/library.rb

```diff
@@ -0,0 +1,53 @@
+require 'date'
+require 'pry'
+
+class Library
+  attr_accessor :books #TODO duedate person status
+
+  def initialize(options={})
+    @books = options[:books] || []
+  end
+
+  def check_in(book)
+    add = {title: book[:title],
+           author: book[:author],
+           status: :okay,
+           person: :non,
+           duedate: " "}
+    @books.push add
+  end
+
+  def lend(user, args={})
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for lend is too high. [22.56/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [15/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)